dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/JSPApplicationTest.java

```diff
@@ -75,7 +80,7 @@ public void jsp_noWebXmlJSP() throws Exception {
         assertTrue(server.isStarted());
 
         String route = Constants.JSP_CONTEXT_ROOT + "/unconfigured.jsp";
-        String expectedRoute = Constants.JSP_CONTEXT_ROOT + "/\\*.jsp";
+        String expectedRoute = Constants.JSP_CONTEXT_ROOT + "/\\*";
```

Comment:
algorithm changed. Matches more with what OTel agent resolves as. *.jsp would have been more unique and something we were capable of doing before, but the backport to JEE7 constrains the information we are able to obtain.